rest/reference.go

```diff
@@ -189,10 +180,7 @@ func (c *ReferenceClient) GetOptionsContract(ctx context.Context, params *models
 //
 // This method returns an iterator that should be used to access the results via this pattern:
 //
-//	iter, err := c.ListConditions(context.TODO(), params, opts...)
-//	if err != nil {
-//		return err
-//	}
+//	iter := c.ListConditions(context.TODO(), params, opts...)
```

Comment:
looks like this is a separate issue but can you fix it? `c.ListConditions` -> `c.ListOptionsContract`